website/docs/r/sm_arbitrary_secret.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_sm_arbitrary_secret" (Beta)
+description: |-
+  Manages ArbitrarySecret.
+subcategory: "IBM Cloud Secrets Manager API"
+---
+
+# ibm_sm_arbitrary_secret
+
+Provides a resource for ArbitrarySecret. This allows ArbitrarySecret to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_sm_arbitrary_secret" {
+  instance_id   = "6ebc4224-e983-496a-8a54-f40a0bfa9175"
+  region        = "us-south"
+  custom_metadata = {"key":"value"}
+  description = "Extended description for this secret."
+  expiration_date = "2022-04-12T23:20:50.520Z"
+  labels = my-label
+  payload = "secret-credentials"
+  secret_group_id = "default"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `custom_metadata` - (Optional, Map) The secret metadata that a user can customize.
```

Comment:
In all resource docs I see all argumets as optional  still the resources marks them as required?

Reply:
Fixed.